src/utils/session.ts

```diff
@@ -189,3 +189,28 @@ export function getUserEmail(session: Session | undefined): string | Error {
         "Unable to find email in session"
     );
 }
+
+/**
+ * Clears journey-specific variables from the session.
+ *
+ * This function resets various session data to ensure a clean slate for the user's journey.
+ * It's particularly useful when a user needs to restart a process or begin a new journey
+ * to prevent any lingering data from causing unexpected behavior.
+ *
+ * @param {Session | undefined} session - The session object to be cleared.
+ * @throws {Error} If the session is undefined.
+ */
+export function clearSession(session: Session | undefined): void {
```

Comment:
You could probably do a quick unit test for this function

Reply:
I've added some unit test for it